tabular/src/autogluon/tabular/models/lgb/hyperparameters/lgb_trial.py

```diff
@@ -15,23 +19,40 @@
 @args()
 def lgb_trial(args, reporter):
     """ Training script for hyperparameter evaluation of Gradient Boosting model """
+
+    model = None
     try:
         model, args, util_args = model_trial.prepare_inputs(args=args)
 
+        epochs = util_args['epochs']
+
+        save_val_pred = util_args.pop('save_val_pred')
```

Comment:
sanity check: pop is modifying underlying dict. Was removing this key intended?

Reply:
Yes. this is intended to avoid downstream functions calls getting unexpected parameters from kwargs